### PR TITLE
Update js `languages`

### PR DESCRIPTION
**Pull request by fisker**
I have a question, why multiple parsers? When will the second one be used?

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: src/language-js/index.js [hunk 1]
```diff
@@ -10,23 +10,23 @@ const languages = [
     require("linguist-languages/data/JavaScript.json"),
     (data) => ({
       since: "0.0.0",
-      parsers: ["babel", "flow"],
+      parsers: ["babel"],
```
Comment (fisker) [inline src/language-js/index.js:13]: I think we should only use `babel`

File: src/language-js/index.js [hunk 2]
```diff
@@ -10,23 +10,23 @@ const languages = [
...
       vscodeLanguageIds: ["javascript", "mongo"],
-      interpreters: data.interpreters.concat(["nodejs"]),
+      interpreters: [...data.interpreters, "nodejs"],
     })
   ),
   createLanguage(require("linguist-languages/data/JavaScript.json"), () => ({
     name: "Flow",
     since: "0.0.0",
-    parsers: ["babel", "flow"],
+    parsers: ["flow", "babel-flow"],
```
Comment (fisker) [inline src/language-js/index.js:21]: Should prefer `flow` parser for `flow`

**Comment by thorn0** [comment]
> I have a question, why multiple parsers? When will the second one be used?

It's up to API consumers to decide how to use this information.

**Comment by fisker** [comment]
Means I can get all possible parsers, and try one by one. In that case, should we list more parsers.

For example, `babel` `flow` `babel-flow` for `js` and `jsx`?

**Comment by thorn0** [comment]
Yes. This might come in handy for example in a Prettier-standalone scenario where only `parser-typescript.js` is loaded.

**Review by thorn0 — APPROVED** [review]

**Review by evilebottnawi — APPROVED** [review]

**Pull request merged**